scripts/support/zephyr_flash_debug.py

```diff
@@ -91,8 +91,15 @@ def main():
     try:
         handlers[args.top_command](args)
     except Exception as e:
-        print('Error: {}'.format(e), file=sys.stderr)
-        raise
+        if args.verbose:
+            raise
+        else:
+            print('Error: {}'.format(e), file=sys.stderr)
+            print(('(Re-run as "zephyr_flash_debug.py --verbose {} ..." '
```

Comment:
Maybe use args[0] here instead of the hardcoded file name?

Reply:
Good point, fixed.